### PR TITLE
Removing dependency on Mandrill in favor of straight SMTP

### LINKED ISSUE #9: Support alternative(s) to Mandrill?
Now that Mandrill is going to require a paid Mailchimp account, any plan to support another api? (Amazon STS, Postmark, SMTP, etc?).

Reference : http://blog.mandrill.com/important-changes-to-mandrill.html

### PR DESCRIPTION
**Pull request by jasondew**
Fixes #9 

/cc @zmoazeni and @tjschuck for some code reviewing

### CODE REVIEW

File: notable.go [hunk 1]
```diff
@@ -2,20 +2,19 @@ package notable
 
 import (
 	"fmt"
-	slack "github.com/harvesthq/notable/Godeps/_workspace/src/github.com/nlopes/slack"
+	slack "github.com/nlopes/slack"
```
Comment (zmoazeni) [inline notable.go]: pretty

**Comment by zmoazeni** [comment]
I honestly have no reference on reviewing this. So +1? Sure. +1

**Comment by tjschuck** [comment]
Like @zmoazeni, RUBBER STAMP :+1: :+1: :+1:

**Comment by tjschuck** [comment]
Also, fixes https://github.com/harvesthq/notable/issues/9, ya?

File: notable.go (hunk 1 above)
Comment (jasondew) [inline notable.go]: Yup. I believe that Heroku used to require that but now [they support it](https://devcenter.heroku.com/articles/go-dependencies-via-godep)

**Comment by jasondew** [comment]
Thanks guys!

**Pull request merged**